devel/git/Portfile

```diff
@@ -214,7 +214,26 @@ variant credential_osxkeychain description {Install git credential-osxkeychain u
     post-destroot {
         xinstall -m 755 "${worksrcpath}/contrib/credential/osxkeychain/git-credential-osxkeychain" \
             "${destroot}${prefix}/libexec/git-core/"
+
+        # Create a global gitconfig file that sets credential.helper to
+        # osxkeychain. This can be overriden on a per-user basis in the user's
+        # own git config file
+        set f [open ${destroot}${prefix}/etc/gitconfig w 0644]
+        puts ${f} "\[credential]\n\thelper = osxkeychain"
```

Comment:
Remember to `close ${f}`.

Reply:
Thanks, this fixes it. I'll create a new PR.